src/pykeen/pipeline.py

```diff
@@ -250,16 +250,150 @@ def title(self) -> Optional[str]:  # noqa:D401
 
-    def plot_losses(self):
+    def plot_losses(self, ax=None):
         """Plot the losses per epoch."""
-        import matplotlib.pyplot as plt
+        if ax is None:
+            import matplotlib.pyplot as plt
+            ax = plt.gca()
+
+        import seaborn as sns
+        sns.set_style('darkgrid')
+
+        rv = sns.lineplot(x=range(len(self.losses)), y=self.losses, ax=ax)
+
+        loss_name = normalize_string(self.model.loss.__class__.__name__, suffix=_LOSS_SUFFIX)
+        ax.set_ylabel(f'{loss_name} Loss')
+        ax.set_xlabel('Epoch')
+        ax.set_title(self.title if self.title is not None else 'Losses Plot')
+        return rv
+
+    def plot_er(  # noqa: C901
+        self,
+        model: Optional[str] = None,
+        margin: float = 0.4,
+        ax=None,
+        entities: Optional[Set[str]] = None,
+        relations: Optional[Set[str]] = None,
+        apply_limits: bool = True,
+        plot_entities: bool = True,
+        plot_relations: bool = False,
+        annotation_x_offset: float = 0.02,
+        annotation_y_offset: float = 0.03,
+        **kwargs,
+    ):
+        """Plot the reduced entities and relation vectors in 2D.
+
+        :param model: The dimensionality reduction model from :mod:`sklearn`. Defaults to PCA.
+            Can also use KPCA, GRP, SRP, TSNE, LLE, ISOMAP, MDS, or SE.
+        :param kwargs: The keyword arguments passed to `__init__()` of
+            the reducer class (e.g., PCA, TSNE)
+
+        .. warning::
+
+            Plotting relations and entities on the same plot is only
+            meaningful for translational distance models like TransE.
+        """
+        if not plot_entities and not plot_relations:
+            raise ValueError
+
+        if model is None:
+            model = 'PCA'
+
+        reducer, reducer_kwargs = _get_model(model, **kwargs)
+
+        if ax is None:
+            import matplotlib.pyplot as plt
+            ax = plt.gca()
+
         import seaborn as sns
-        sns.set()
-        plt.ylabel('Loss')
-        plt.xlabel('Epoch')
-        if self.title is not None:
-            plt.title(self.title)
-        return sns.lineplot(x=range(len(self.losses)), y=self.losses)
+        sns.set_style('whitegrid')
+
+        # reduce entity embedding dimensionality
+        e_emb = self.model.entity_embeddings.weight.detach().numpy()
+        if e_emb.shape[1] != 2:
+            entity_reduction_model = reducer(n_components=2, **reducer_kwargs)
+            e_emb_red = entity_reduction_model.fit_transform(e_emb)
+        else:
+            logger.debug('not reducing entity embeddings, already dim=2')
+            e_emb_red = e_emb
+
+        # reduce relation embedding dimensionality
+        r_emb = self.model.relation_embeddings.weight.detach().numpy()
+        if r_emb.shape[1] != 2:
+            relation_reduction_model = reducer(n_components=2, **reducer_kwargs)
+            r_emb_red = relation_reduction_model.fit_transform(r_emb)
```

Comment:
The relation reduction method should be the same as for the entities, otherwise we might end up with non-aligned projections.

Reply:
Also, could we error out in case we want to plot relations, but do not use a linear projection?

Reply:
I'm not sure what you mean by the first comment - the entities and relations will always be reduced the same way. I think the second idea is good and I will take care of that

Reply:
In 61a1e688 I made it so if you don't use PCA when trying to plot relations, it throws a `ValueError`. Were there any other models that should also be considered valid?

Reply:
For the first comment, do you mean that the reducer that's fit on the entities should be applied to the relations?

Reply:
> In [61a1e68](https://github.com/pykeen/pykeen/commit/61a1e688df2c6e4f49657b0e4dd7105d73117cba) I made it so if you don't use PCA when trying to plot relations, it throws a `ValueError`. Were there any other models that should also be considered valid?

I don't think other ones are applicable.



> For the first comment, do you mean that the reducer that's fit on the entities should be applied to the relations?

If you plot the entities and relations in the same coordinate system their axes should match. If we have a separate reducer for the relations, the PCA fit might find different principal components than for the entities, meaning that the axes of the projections are not aligned with each other.

Reply:
Okay that makes sense. Done in e342b048